### PR TITLE
Short cuts addendum

### PR DESCRIPTION
**Pull request by mbgower**
Edits for clarity, sytle and readability. Also incorporated the remaining comments from https://github.com/w3c/wcag/issues/351

### CODE REVIEW

**Comment by mbgower** [comment]
This Understanding doc could still use a bit of TLC, but I think this helps push it in the right direction.

File: understanding/21/character-key-shortcuts.html [hunk 1]
```diff
@@ -9,43 +9,39 @@
 		<h1>Understanding Character Key Shortcuts</h1>
 		<section id="intent">
 			<h2>Intent</h2>
-<p>Character key shortcuts work well for many keyboard users, but are inappropriate and frustrating for speech input users &#8212; whose means of input is strings of letters &#8212; and for keyboard users who are prone to accidentally hitting keys.
-As a result, users must be able to turn off or reconfigure shortcuts made up of a single character key, or two or more successive character keys.
+<p>The intent of this Success Crition is to reduce accidental activation of keyboard shortcuts. Character key shortcuts work well for many keyboard users, but are inappropriate and frustrating for speech input users &#8212; whose means of input is strings of letters &#8212; and for keyboard users who are prone to accidentally hit keys.
+To rectify this issue, authors need to allow users to turn off or reconfigure shortcuts that are made up of only one or more character keys.
```
Comment (awkawk) [inline understanding/21/character-key-shortcuts.html]: "made up of only character keys." ?

Comment (mbgower) [reply understanding/21/character-key-shortcuts.html]: "made up of only character keys" works for me

File: understanding/21/character-key-shortcuts.html [hunk 2]
```diff
@@ -9,43 +9,39 @@
...
+<p>For example, a speech-input user named Kim has her cursor focus in the main window of a web mail application that uses common keyboard shortcuts to navigate ("k"), archive ("y") and mute messages ("m"). A coworker named Mike enters her office and says "Hey Kim" and her microphone picks that up. The Y of "hey" archives the current message. K in "Kim" moves down one conversation and M mutes a message or thread. And, if Kim looks up and says "Hey Mike" without remembering to turn off the microphone, the same three things happen in a different sequence.</p>
+<p>A user interacting with a webpage or web app that doesn't use single-character shortcuts doesn't have this problem. Inadvertent strings of characters from the speech application lack the modifier key and are not interpreted as shortcuts. A speech user filling in a text input form may find that a phrase that is accidentally picked up by the speech microphone results in stray text being entered into the field, but that is easily seen and undone. The Resources section of this page contains links to videos demonstrating these types of issues.</p>
```
Comment (alastc) [inline understanding/21/character-key-shortcuts.html]: Slight non-sequeter in the second sentence, how about:
Inadvertent strings of characters from the speech application are not interpreted as shortcuts if a modifer key is required.

File: understanding/21/character-key-shortcuts.html (hunk 1 above)
Comment (mbgower) [reply understanding/21/character-key-shortcuts.html]: adopted in PR

File: understanding/21/character-key-shortcuts.html (hunk 2 above)
Comment (mbgower) [reply understanding/21/character-key-shortcuts.html]: adopted

**Pull request merged**